assets/queries/k8s/service_account_key_file_not_properly_set/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "dab4ec72-ce2e-4732-b7c3-1757dcce01a1",
+  "queryName": "Service Account Key File Not Properly Set",
+  "severity": "MEDIUM",
+  "category": "Secret Management",
+  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be difined and set a PEM encoded file",
```

Misspelling the word defined
```suggestion
  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be defined and set a PEM encoded file",
```